### PR TITLE
Adding changes for CV notebook

### PR DESCRIPTION
**Pull request by mrinal18**
@sergeyplis  @Scitator Please review the notebooks for computer vision

### CODE REVIEW

**Review by Scitator — CHANGES_REQUESTED** [review]
`Deep_Learning_Course_Chapter_7.ipynb`
- let's fix a bit
```
assert prediction.detach().cpu()..
->
assert prediction["logits"].detach().cpu()..
```
- could you please specify all init args for the `dl.SupervisedRunner()`?
	- I think, it would be much easier to understand it than
- the same for `runner = SupervisedRunner()`
- btw, could you please specify `valid_loader="valid", "valid_metric=accuracy01"` for "Implement a classification of cats and dogs using catalyst without any arguments."?
- could you please also show the tensorboard plots with the logs?
- if you have too much metrics, please use `verbose=False` for simplified logging

PS. should we also review `Introduction_to_convnet.ipynb` notebook?

**Comment by mrinal18** [comment]
@Scitator thanks a lot! i am looking into this right now and will update it accordingly!! 

yes,can you also look at Introduction_to_convnet.ipynb and let me know?

**Comment by Scitator** [comment]
- not sure why, but couldn't load the figures from the notebook
- `pip install -U torch==1.8.0 torchvision==0.9.0 catalyst==21.04.2 albumentations==0.5.0` - let's update dependencies to the last one
	- btw, as far as you are using `torchvision`, could you please use torchvision transforms? Catalyst has a few for examples puproses, nevertheless it would be more convenient to use `torchvision` variant
- could you please clean unused dependencies, like `from torchvision.datasets import CIFAR10`
	- you could open the notebook in vscode for that
- `callbacks = [AccuracyCallback(num_classes=2, accuracy_args=[1])]`
	- should be `dl.AccuracyCallback` I think
- btw, don't you mind to add a short introduction about which metric names to expect during run with callbacks?
	- like their names? `precision, recall, f1, support` and aggregations `class_{}, _micro, _macro, _weighted`

**Comment by mrinal18** [comment]
@Scitator can you review the changes made? 
note: i will add few more points in the metrics using catalyst part

**Comment by Scitator** [comment]
`Introduction_to_convnet.ipynb`:
- still could not check the figures from notebooks - they are just not loading at all 😢 
- and let's fix the versions :top: - it could be important in a year
  - and use `torchvision` transforms
- btw, could you please provide a colab version of this notebook? so it could be done within the colab

> Also, it is a good practice to inherit from dl.Runner and implement the run_train_epoch and run_valid_epoch functions.
- not sure about the `run_train_epoch` and `run_valid_epoch` :)


`UnderstandingMetrics.ipynb`:
- looks really well done, should be expect a few Catalyst examples?

**Comment by mrinal18** [comment]
> 

Thanks for the review! 

1. not sure why the notebook figures is not loading properly :/ let me fix that asap.
2. will change the version part, i thought i already did but might have missed it 
3. aah, yes i am planning to shift everything to colab (i did for NLP notebooks) 
4. thanks for the metrics review! and yes, there will be a few Catalyst examples :)
